content/en/blog/_posts/2025-05-22-gateway-api-v1.3/index.md

```diff
@@ -0,0 +1,370 @@
+  - group: gateway.networking.k8s.io
+    kind: Gateway
+    name: http-gateway
+  rules:
+  - backendRefs:
+    - kind: Service
+      name: http-route-cors
+      port: 80
+    matches:
+    - path:
+        type: PathPrefix
+        value: /resource/foo
+    filters:
+    - cors:
+      - allowOrigins:
+        - *
+        allowMethods: 
+        - GET
+        - HEAD
+        - POST
+        allowHeaders: 
+        - Accept
+        - Accept-Language
+        - Content-Language
+        - Content-Type
+        - Range
+      type: CORS
+```
+In this case, the `Gateway` returns an origin header of "*", which means that the
+requested resource can be accessed from any `Origin`, a methods header consisting
+of `GET`, `HEAD`, and `POST`, and a headers header of `Accept`, `Accept-Language`,
+`Content-Language`, `Content-Type`, and `Range`.
+
+```text
+HTTP/1.1 200 OK
+Access-Control-Allow-Origin: *
+Access-Control-Allow-Methods: GET, HEAD, POST
+Access-Control-Allow-Headers: Accept,Accept-Language,Content-Language,Content-Type,Range
+```     
+The complete list of fields in the new CORS filter:
+* AllowOrigins
+* AllowMethods
+* AllowHeaders
+* AllowCredentials
+* ExposeHeaders
+* MaxAge
+
+See [CORS protocol](https://fetch.spec.whatwg.org/#http-cors-protocol) for details.
+
+### XListenerSets (Standard Mechanism for Listener  and Gateway Merging)
+Lead: [Dave Protasowski](https://github.com/dprotaso)
+
+GEP-1713: [ListenerSets - Standard Mechanism to Merge Multiple Gateways](https://github.com/kubernetes-sigs/gateway-api/pull/3213)
+
+This feature adds a new experimental resource called `XListenerSet` to allow a
+shared list of Listeners to be attached to one or more parent Gateway(s).  In
+addition, it expands upon the current suggestion that implementations may merge
+`Gateway` resources.  It also:
+
+- adds new field `AllowedListeners` to `GatewaySpec`. `AllowedListeners` defines
+from which namespaces to select XListenerSets that are allowed to attach to that
+`Gateway` (Same, All, None, or Selector based)
+- renames the `Listener` resource to `ListenerEntry`
+- re-types the `Port` resource to allow for dynamic port assignment
+- adds a new `Gateway` condition type `AttachedListenerSets`
+- provides for xRoute `parentRefs` to contain both `XListenerSet` and `Gateway`
+
+The following example shows a `Gateway` with an HTTP listener and two child HTTPS
+XListenerSets with unique hostnames and certificates.
+
+```yaml
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: parent-gateway
+  namespace: parent-listener
+spec:
+  gatewayClassName: example
+  allowedListeners:
+  - from: Same
+  listeners:
+  - name: foo
+    hostname: foo.com
+    protocol: HTTP
+    port: 80
+---
+apiVersion: gateway.networking.x-k8s.io/v1alpha1
+kind: XListenerSet
+metadata:
+  name: first-workload-listeners
+  namespace: parent-listener
+spec:
+  parentRef:
+    name: parent-gateway
+    kind: Gateway
+    group: gateway.networking.k8s.io
```

Comment:
I _think_ both of these are optional, recommend removing throughout for a more concise example.
```suggestion
```